man/rgblink.5

```diff
@@ -1,53 +1,37 @@
 .\" SPDX-License-Identifier: MIT
 .\"
-.Dd March 28, 2021
+.Dd December 22, 2023
 .Dt RGBLINK 5
 .Os
 .Sh NAME
 .Nm rgblink
 .Nd linker script file format
 .Sh DESCRIPTION
-The linker script is an external file that allows the user to specify the order of sections at link time and in a centralized manner.
+The linker script is a self-contained file that allows specifying attributes for sections at link time, and in a centralized manner.
```

Comment:
It's not self-contained if it has `INCLUDE`s, and tbh I'm not sure what property of linker scripts that's meant to convey anyway.
```suggestion
The linker script is a file that allows specifying attributes for sections at link time, and in a centralized manner.
```